### PR TITLE
updated docs to add some missing data

### PR DESCRIPTION
**Pull request by dneuman64**

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/949/
Test PASSed.

File: docs/source/admin/traffic_ops/using.rst [hunk 1]
```diff
@@ -293,135 +293,136 @@ The fields in the Delivery Service view are:
...
++---------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Info URL                              | Info URL  for this delivery service. To be consumed from the APIs by downstream tools (Portal).                                                                                                                   |
++---------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Initial Dispersion                    | For HTTP delivery services - the number of caches that will be used by Traffic Router.                                                                                                                            |
```
Comment (limited) [inline docs/source/admin/traffic_ops/using.rst]: Suggest to add something like "If caches reach capacity, additional caches will be added as needed"

File: docs/source/admin/traffic_ops/using.rst [hunk 2]
```diff
@@ -293,135 +293,136 @@ The fields in the Delivery Service view are:
...
++---------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Key (XML ID)                          | A unique string that identifies this delivery service.                                                                                                                                                            |
++---------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Logs Enabled?                         | Whether or not Traffic Logs is enabled for the delivery service.  This feature is currently imcomplete without Traffic Logs                                                                                       |
```
Comment (limited) [inline docs/source/admin/traffic_ops/using.rst]: typo 'imcomplete'

File: docs/source/admin/traffic_ops/using.rst [hunk 3]
```diff
@@ -293,135 +293,136 @@ The fields in the Delivery Service view are:
...
+|                                       |                                                                                                                                                                                                                   |
+|                                       | **Note:** you will not be able to save a Regex Remap Expression if you have Query String Handling set to drop query strings at the edge. See :ref:`rl-regex-remap`.                                               |
++---------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Regional Geo-Blocking                 |                                                                                                                                                                                                                   |
```
Comment (limited) [inline docs/source/admin/traffic_ops/using.rst]: Can you please add a link to `docs/source/admin/quick_howto/regionalgeo.rst` here?

Comment (dneuman64) [reply docs/source/admin/traffic_ops/using.rst]: done

File: docs/source/admin/traffic_ops/using.rst (hunk 2 above)
Comment (dneuman64) [reply docs/source/admin/traffic_ops/using.rst]: done

File: docs/source/admin/traffic_ops/using.rst (hunk 1 above)
Comment (dneuman64) [reply docs/source/admin/traffic_ops/using.rst]: done

**Comment by dneuman64** [comment]
@limited can you take a look again?

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/998/
Test FAILed.

**Pull request merged**